README.md

```diff
@@ -1 +1,97 @@
-# service-qiskit
\ No newline at end of file
+# service-qiskit
+
+This is an Open Horizon services configuration to deploy a docker container with QISKIT and Jupyter for Raspberry Pi.
+
+QISKIT is both a quantum simulator and a tool that simplifies interfacing with IBM's quantum computing services in the IBM cloud. The Quantum Experience link below will guide you through the latter if you are interested in using the real thing!
+
+There's a blog article to go along with this repo, here: 
+[https://darlingevil.com/your-own-quantum-computer/](https://darlingevil.com/your-own-quantum-computer/)
+
+This container was built to run on a Raspberry Pi 4B with 2GB RAM (with Raspberry Pi OS 10, buster). It should work on any Raspberry Pi model 2, 3, or 4, although you may need to expland swap space on the smaller machines as described below. It should also work on any 64-bit x86 host with Docker installed.
+
+## Prerequisites
+
+You need to install to use this container is docker. You can install docker on a Raspberry Pi with this one command:
+
+```
+curl -sSL https://get.docker.com | sh
+```
+
+When done, I recommend running this command so the pi user can use docker withut sudo:
+
+```
+sudo usermod -aG docker pi
+```
+
+After executing that command, exit your shell and open a new shell. In that new shell and all subsequent shells you will be able to run `docker` commands without `sudo`. E.g., try this:
+
+```
+docker ps
+```
+
+## To build this container
+
+NOTE: You need a little more than 2GB of RAM to build the docker container (about 3.4GB I think).  Note also that this extra memory is not needed to **run** the container, only to **build** it. A 4GB or 8GB Pi therefore won't need this step so don't bother with it. However, since I am using a 2GB Raspberry Pi 4B, I increased the swap space with these commands:
+
+```
+sudo sed -i 's/CONF_SWAPSIZE=100/CONF_SWAPSIZE=1024/' /etc/dphys-swapfile
+sudo /etc/init.d/dphys-swapfile stop
+sudo /etc/init.d/dphys-swapfile start
+```
+
+Once you have that sorted out, these are the build steps:
+
+1. Edit the Makefile to set your DockerHub ID in `DOCKERHUB_ID`
```

Comment:
Looks like this is the only other mention of `DOCKERHUB_ID` outside of the Makefile